lib/service-data/service-data.js

```diff
@@ -297,12 +297,23 @@ external.getInstanceIdByPropertyValue = (property, value) => {
   return instance._id
 }
 
-external.getInstanceTitle = (_id) => {
+external.getInstanceTitle = (_id, isSummary = false) => {
   const instance = external.getInstance(_id)
+
   if (!instance) {
     return
   }
-  let title = instance.title || instance.heading || instance.legend || instance.label || instance.text
+
+  let title
+
+  if (isSummary) {
+    title = instance.legendSummary || instance.labelSummary || instance.textSummary
```

Comment:
Aren't we missing `headingSummary` here?

Reply:
Don't think that uses this function? https://github.com/ministryofjustice/fb-runner-node/blob/4a341ef2dc77555c74556660d87c2a545042e4a3/lib/page/controller/page.summary/page.summary.controller.js#L156 but might have misunderstood

Reply:
Similary, wouldn't this be better as

```
let pageHeading = getInstanceTitleSummary(stepInstance._id)
```

Reply:
Done